setup.py

```diff
@@ -2,15 +2,15 @@
 from setuptools import setup
 from setuptools import find_packages
 
-VERSION = '0.23.0'
+VERSION = '0.24.0'
 
 AUTHORS = 'Matthew Bourque, Lauren Chambers, Misty Cracraft, Mike Engesser, Mees Fix, Joe Filippazzo, Bryan Hilbert, '
 AUTHORS += 'Graham Kanarek, Teagan King, Catherine Martlin, Maria Pena-Guerrero, Johannes Sahlmann, Ben Sunnquist'
 
 DESCRIPTION = 'The James Webb Space Telescope Quicklook Project'
 
-DEPENDENCY_LINKS = ['git+https://github.com/spacetelescope/jwst#0.16.2',
-                    'git+https://github.com/spacetelescope/jwst_reffiles'
+DEPENDENCY_LINKS = ['git+https://github.com/spacetelescope/jwst@0.16.2',
```

Comment:
Should we change this to the pypi version now?

Reply:
Good idea!  I'll put that change in